schema/schema_template.py

```diff
@@ -97,8 +97,8 @@ class PhotometryFilters(Base):
         ForeignKey("Telescopes.telescope", onupdate="cascade"),
         primary_key=True,
     )
-    effective_wavelength = Column(Float, nullable=False)
-    width = Column(Float)
+    effective_wavelength_micron = Column(Float, nullable=False)
+    width_micron = Column(Float)
```

Comment:
For the template, I would recommend not specifying unit in the Photometry table since its one I can see different use cases wanting different units. The column name would be vague and the ingest script could have an input that specifies what the unit is and/or let that be something that's individually implemented.

Reply:
done!